### DISCUSSION TITLE
Supply Chain Attacks

### DISCUSSION THREAD

**Discussion by Lerch2021** [Security-Onion-Solutions/securityonion#4908] [Unsupported Versions] [answered]
I have a question about Security Onion that may be difficult to answer.

How does Security Onion prevent supply chain attacks in their own software? 
I.E. if something like a new version of Suricata or Elasticsearch is released with a backdoor or a trojan (picture something like Solarwinds) and then that update for Suricata or Elasticsearch is then rolled up into the next version of Security Onion and released to the public. What kind of tools or mitigation procedures does Security Onion have for a supply chain attack like this? Do they test all open source 3rd party, 4th party, etc code that gets included in Security Onion?

Thank you for your time

**Comment by dougburks** [comment, marked as answer]
Hi @Lerch2021 ,

We've updated our docs to address this:
https://securityonion.net/security#product-and-supply-chain-integrity
